client/my-sites/patterns/components/header/index.tsx

```diff
@@ -1,23 +1,207 @@
+import { Substitution, translate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
 import { preventWidows } from 'calypso/lib/formatting';
 import { PatternsSearchField } from 'calypso/my-sites/patterns/components/search-field';
+import { usePatternsContext } from 'calypso/my-sites/patterns/context';
+import type { Category } from 'calypso/my-sites/patterns/types';
 
 import './style.scss';
 
-type PatternsHeaderProps = {
-	description: string;
-	title: string;
+const CONTENT: Record<
+	Category[ 'name' ],
+	{ description: Substitution; heading: Substitution; title: Substitution }
+> = {
+	default: {
+		title: translate( 'WordPress Patterns', {
+			comment: 'HTML title of the Pattern Library home page',
+		} ),
+		heading: translate( "It's easier with patterns", {
+			comment: 'Heading of the Pattern Library home page',
+		} ),
+		description: translate(
+			'Dive into hundreds of expertly designed, fully responsive layouts, and bring any kind of site to life, faster.',
+			{ comment: 'Intro text on the Pattern Library home page' }
+		),
+	},
+	header: {
+		title: translate( 'WordPress Header Patterns', {
```

Comment:
```suggestion
		title: translate( 'Header WordPress Patterns', {
```
I checked the `/themes` and `/plugins` page. The convention is to start with the category, e.g. "Blog WordPress Themes".

Reply:
That makes sense from an SEO perspective 👍

Reply:
There wouldn't be much of an SEO difference here, but sticking with the established naming conventions is a fine idea. 👍